frontend/src/components/housing/submission/SubmissionAssistance.vue

```diff
@@ -46,44 +43,19 @@ const checkApplicantValuesValid = (
   return isValid;
 };
 
-const confirmSubmit = (data: any) => {
+const confirmSubmit = () => {
   confirm.require({
-    message: 'Are you sure you wish to submit this form? Please review the form before submitting.',
-    header: 'Please confirm submission',
+    message: 'Are you sure you wish for assistance with this form? Please review the form before submitting.',
```

Comment:
English: Something feels off with this sentence... would be using a more active/direct tense to say something along the lines of
```txt
Are you sure about requesting assistance with this form? Please review the form before submitting.
```

Reply:
Consulting with Subin for better wording now. Thanks for pointing it out.

Reply:
totally missed it, it does sound awkward.. I like what you suggested, or .. we can do something like "Are you sure you wish to submit this form for assistance? Please review the form before submitting."